lib/core.ps1

```diff
@@ -698,7 +698,7 @@ function env($name, $global, $val = '__get') {
         $RegistryValueOption = [Microsoft.Win32.RegistryValueOptions]::DoNotExpandEnvironmentNames
         $EnvRegisterKey.GetValue($name, $null, $RegistryValueOption)
     } elseif ($val -eq $null) {
-        $EnvRegisterKey.DeleteValue($name)
+        try { $EnvRegisterKey.DeleteValue($name) } catch { }
```

Comment:
Operation of adding, deleting environment variable both no output, so a empty catch block here.